lib/improver/tests/wind_calculations/wind_gust_diagnostic/test_WindGustDiagnostic.py

```diff
@@ -323,16 +323,6 @@ def test_raises_error_points_mismatch_and_bounds(self):
         with self.assertRaisesRegex(ValueError, msg):
             plugin.process(cube_wg, self.cube_ws)
 
-    def test_raises_error_points_mismatch_and_invalid_bounds(self):
```

Comment:
Removed because there is no reasonably conceivable situation in which a cube would have a coordinate with bounds on some points and not others.